snapshots/hmd/2022-12-07/hmd.py

```diff
@@ -0,0 +1,41 @@
+"""Get HMD full dataset into Snapshots.
```

Comment:
I think we should add something like this as template in walkthrough, for the cases when the file needs to be added manually @Marigold .